### DISCUSSION TITLE
[Script request] Music Assistant

### DISCUSSION THREAD

**Discussion by HeedfulCrayon** [community-scripts/ProxmoxVE#681] [Request script] [answered]
### Application Name

Music Assistant

### Website

https://music-assistant.io

### Description

Music Assistant is a music library manager for your offline and online music sources which can easily stream your favourite music to a wide range of supported players and be combined with the power of Home Assistant!

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by HeedfulCrayon** [comment]
This installation process seems to work well.  Could be a good jumping off point to create the script
https://www.reddit.com/r/homeassistant/comments/1finyrm/comment/lnn7t7x/

**Comment by MickLesk** [comment]
But are you aware that the thing simply can't do anything without Snapcast and co? So it can't even play music on the local device. My feature request is one of the most voted but the dev's don't do anything with it.

**Reply by HeedfulCrayon** [reply]
Playing on my local device would be cool, but I definitely find it useful when I have a house full of google speakers and several different music services I use

**Comment by Happy-Cadaver** [comment]
+1 for this script request.

**Comment by oOStroudyOo** [comment]
Do you have Home Assistant?
If so, Music Assistant is now an official Add-on for it.

**Reply by HeedfulCrayon** [reply]
Won't work for my use case.  I need music assistant to be running on my IoT vlan, and home assistant doesn't allow you to specify which interface an add-on uses

**Reply by oOStroudyOo** [reply]
Fair enough.
I'm sure somebody may be able to pick this up and get it into a helper script.

It may also be worth putting that feature request in on the home/music assistant forums. I'm sure you're not the only person in the world that would appreciate being able to specify interfaces for HA add-ons.

**Reply by HeedfulCrayon** [reply]
@oOStroudyOo Good point, I will submit a request there as well!

**Reply by HeedfulCrayon** [reply]
@oOStroudyOo Requested https://github.com/orgs/music-assistant/discussions/3392

**Comment by MickLesk** [comment]
Dudes, Push this Requests :D 

https://github.com/orgs/music-assistant/discussions/2347

**Comment by TheMagnetar** [comment]
This is being implemented: https://github.com/music-assistant/server/pull/2043

**Reply by MickLesk** [reply]
But not released

**Reply by Happy-Cadaver** [reply]
Now released :) [Music Assistant 2.5.0](https://github.com/music-assistant/server/releases/tag/2.5.0)

**Reply by MickLesk** [reply]
Then maybe in next Release Pipeline (May/June)

**Comment by MickLesk** [comment]
Ready for Testing:
https://github.com/community-scripts/ProxmoxVED/issues/183

Comments in the Issue Section, please

**Comment by MickLesk** [comment, marked as answer]
Bad news, the devs don't support our plan at all. It was treated rather negatively. To be honest, I had never received such a negative response to a request. Basically, the script was finished except the Airplay Support, but they didn't address my issue at all and just condemned everything. I don't see any future in it in terms of the whole contributor. Don't get mad at us, but I don't have the time or inclination to deal with it.

We are removing the test script and rejecting musicassistant.

https://github.com/music-assistant/support/discussions/3879#discussioncomment-12844543

**Reply by Happy-Cadaver** [reply]
Bummer, but I understand.

**Reply by milandzuris** [reply]
> Zlá správa, vývojári vôbec nepodporujú náš plán. Bol prijatý dosť negatívne. Úprimne povedané, nikdy som nedostal takú negatívnu odpoveď na žiadosť. V podstate bol skript dokončený okrem Airplay Support, ale vôbec sa na môj problém nevenovali a všetko len odsúdili. Nevidím v tom žiadnu budúcnosť z hľadiska celého prispievateľa. Nehnevajte sa na nás, ale nemám čas ani chuť sa tým zaoberať.
> 
> Odstraňujeme testovací skript a zamietame musicassistant.
> 
> [hudobný asistent/podpora #3879 (komentár)](https://github.com/orgs/music-assistant/discussions/3879#discussioncomment-12844543)

Sorry for question but something new about Music Assistant work with LXC?, thx.
